stdlib/sql/from.go

```diff
@@ -4,15 +4,15 @@ import (
 	"context"
 	"database/sql"
 
-	_ "github.com/go-sql-driver/mysql"
+	_ "github.com/lib/pq"
+
```

Comment:
fix imports.

You should only separate stdlib VS others.
Don't know if this is widespread, I know this is our practice :)